----------------------------------------------------------------
Sun Apr 28 15:10:01 EDT 2019:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.10.2.0 - (1582446): instance a816c00e-016a-655a-a966-000006a0f960 
on database directory H:\git\qherb-pnelson1-wwyatt-jbady\EntreLink\test.db with class loader sun.misc.Launcher$AppClassLoader@4e25154f 
Loaded from file:/H:/CS320/Workspace/CS320_Derby/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_181-b13
user.dir=H:\git\qherb-pnelson1-wwyatt-jbady\EntreLink
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null

------------  Begin Shutdown Error Stack -------------

ERROR XSLA0: Cannot flush the log file to disk H:\git\qherb-pnelson1-wwyatt-jbady\EntreLink\test.db\log\log1.dat.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseDataFileFactory.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.store.raw.data.RAFContainer.stubbify(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseContainerHandle.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerOperation.undoMe(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerUndoOperation.doMe(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.logAndUndo(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.logAndUndo(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.undo(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.abort(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.XactFactory.rollbackAllTransactions(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.recover(Unknown Source)
	at org.apache.derby.impl.store.raw.RawStore.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.store.access.RAMAccessManager.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.bootStore(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startProviderService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.findProviderAndStartService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startPersistentService(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.startPersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at edu.ycp.cs320.sqldemo.SQLDemo.main(SQLDemo.java:43)
Caused by: java.io.IOException: Access is denied
	at java.io.RandomAccessFile.writeBytes(Native Method)
	at java.io.RandomAccessFile.write(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.writeToLog(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.flushDirtyBuffers(Unknown Source)
	... 45 more
============= begin nested exception, level (1) ===========
java.io.IOException: Access is denied
	at java.io.RandomAccessFile.writeBytes(Native Method)
	at java.io.RandomAccessFile.write(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.writeToLog(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.flushDirtyBuffers(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseDataFileFactory.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.store.raw.data.RAFContainer.stubbify(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseContainerHandle.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerOperation.undoMe(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerUndoOperation.doMe(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.logAndUndo(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.logAndUndo(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.undo(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.abort(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.XactFactory.rollbackAllTransactions(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.recover(Unknown Source)
	at org.apache.derby.impl.store.raw.RawStore.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.store.access.RAMAccessManager.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.bootStore(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startProviderService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.findProviderAndStartService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startPersistentService(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.startPersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at edu.ycp.cs320.sqldemo.SQLDemo.main(SQLDemo.java:43)
============= end nested exception, level (1) ===========


------------  End Shutdown Error Stack -------------

----------------------------------------------------------------
Sun Apr 28 15:10:02 EDT 2019:
Shutting down instance a816c00e-016a-655a-a966-000006a0f960 on database directory H:\git\qherb-pnelson1-wwyatt-jbady\EntreLink\test.db with class loader sun.misc.Launcher$AppClassLoader@4e25154f 
Sun Apr 28 15:10:02 EDT 2019 Thread[main,5,main] Cleanup action starting
java.sql.SQLException: Failed to start database 'test.db' with class loader sun.misc.Launcher$AppClassLoader@4e25154f, see the next exception for details.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at edu.ycp.cs320.sqldemo.SQLDemo.main(SQLDemo.java:43)
Caused by: java.sql.SQLException: Failed to start database 'test.db' with class loader sun.misc.Launcher$AppClassLoader@4e25154f, see the next exception for details.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 13 more
Caused by: java.sql.SQLException: An exception was thrown during transaction abort.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	... 10 more
Caused by: ERROR XSTB0: An exception was thrown during transaction abort.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.abort(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.XactFactory.rollbackAllTransactions(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.recover(Unknown Source)
	at org.apache.derby.impl.store.raw.RawStore.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.store.access.RAMAccessManager.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.bootStore(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startProviderService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.findProviderAndStartService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startPersistentService(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.startPersistentService(Unknown Source)
	... 10 more
Caused by: ERROR XSLA8: Cannot rollback transaction 418, trying to compensate null operation with null
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.undo(Unknown Source)
	... 32 more
Caused by: ERROR XSLA1: Log Record has been sent to the stream, but it cannot be applied to the store (Object null).  This may cause recovery problems also.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.logAndUndo(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.logAndUndo(Unknown Source)
	... 33 more
Caused by: ERROR XSLA0: Cannot flush the log file to disk H:\git\qherb-pnelson1-wwyatt-jbady\EntreLink\test.db\log\log1.dat.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseDataFileFactory.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.store.raw.data.RAFContainer.stubbify(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseContainerHandle.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerOperation.undoMe(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerUndoOperation.doMe(Unknown Source)
	... 35 more
Caused by: java.io.IOException: Access is denied
	at java.io.RandomAccessFile.writeBytes(Native Method)
	at java.io.RandomAccessFile.write(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.writeToLog(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.flushDirtyBuffers(Unknown Source)
	... 45 more
============= begin nested exception, level (1) ===========
java.sql.SQLException: An exception was thrown during transaction abort.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at edu.ycp.cs320.sqldemo.SQLDemo.main(SQLDemo.java:43)
Caused by: java.sql.SQLException: An exception was thrown during transaction abort.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 12 more
Caused by: ERROR XSTB0: An exception was thrown during transaction abort.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.abort(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.XactFactory.rollbackAllTransactions(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.recover(Unknown Source)
	at org.apache.derby.impl.store.raw.RawStore.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.store.access.RAMAccessManager.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.bootStore(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startProviderService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.findProviderAndStartService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startPersistentService(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.startPersistentService(Unknown Source)
	... 10 more
Caused by: ERROR XSLA8: Cannot rollback transaction 418, trying to compensate null operation with null
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.undo(Unknown Source)
	... 32 more
Caused by: ERROR XSLA1: Log Record has been sent to the stream, but it cannot be applied to the store (Object null).  This may cause recovery problems also.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.logAndUndo(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.logAndUndo(Unknown Source)
	... 33 more
Caused by: ERROR XSLA0: Cannot flush the log file to disk H:\git\qherb-pnelson1-wwyatt-jbady\EntreLink\test.db\log\log1.dat.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseDataFileFactory.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.store.raw.data.RAFContainer.stubbify(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseContainerHandle.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerOperation.undoMe(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerUndoOperation.doMe(Unknown Source)
	... 35 more
Caused by: java.io.IOException: Access is denied
	at java.io.RandomAccessFile.writeBytes(Native Method)
	at java.io.RandomAccessFile.write(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.writeToLog(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.flushDirtyBuffers(Unknown Source)
	... 45 more
============= end nested exception, level (1) ===========
============= begin nested exception, level (2) ===========
java.sql.SQLException: An exception was thrown during transaction abort.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at edu.ycp.cs320.sqldemo.SQLDemo.main(SQLDemo.java:43)
Caused by: ERROR XSTB0: An exception was thrown during transaction abort.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.abort(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.XactFactory.rollbackAllTransactions(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.recover(Unknown Source)
	at org.apache.derby.impl.store.raw.RawStore.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.store.access.RAMAccessManager.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.bootStore(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startProviderService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.findProviderAndStartService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startPersistentService(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.startPersistentService(Unknown Source)
	... 10 more
Caused by: ERROR XSLA8: Cannot rollback transaction 418, trying to compensate null operation with null
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.undo(Unknown Source)
	... 32 more
Caused by: ERROR XSLA1: Log Record has been sent to the stream, but it cannot be applied to the store (Object null).  This may cause recovery problems also.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.logAndUndo(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.logAndUndo(Unknown Source)
	... 33 more
Caused by: ERROR XSLA0: Cannot flush the log file to disk H:\git\qherb-pnelson1-wwyatt-jbady\EntreLink\test.db\log\log1.dat.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseDataFileFactory.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.store.raw.data.RAFContainer.stubbify(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseContainerHandle.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerOperation.undoMe(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerUndoOperation.doMe(Unknown Source)
	... 35 more
Caused by: java.io.IOException: Access is denied
	at java.io.RandomAccessFile.writeBytes(Native Method)
	at java.io.RandomAccessFile.write(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.writeToLog(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.flushDirtyBuffers(Unknown Source)
	... 45 more
============= end nested exception, level (2) ===========
============= begin nested exception, level (3) ===========
ERROR XSTB0: An exception was thrown during transaction abort.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.abort(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.XactFactory.rollbackAllTransactions(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.recover(Unknown Source)
	at org.apache.derby.impl.store.raw.RawStore.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.store.access.RAMAccessManager.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.bootStore(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startProviderService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.findProviderAndStartService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startPersistentService(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.startPersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at edu.ycp.cs320.sqldemo.SQLDemo.main(SQLDemo.java:43)
Caused by: ERROR XSLA8: Cannot rollback transaction 418, trying to compensate null operation with null
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.undo(Unknown Source)
	... 32 more
Caused by: ERROR XSLA1: Log Record has been sent to the stream, but it cannot be applied to the store (Object null).  This may cause recovery problems also.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.logAndUndo(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.logAndUndo(Unknown Source)
	... 33 more
Caused by: ERROR XSLA0: Cannot flush the log file to disk H:\git\qherb-pnelson1-wwyatt-jbady\EntreLink\test.db\log\log1.dat.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseDataFileFactory.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.store.raw.data.RAFContainer.stubbify(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseContainerHandle.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerOperation.undoMe(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerUndoOperation.doMe(Unknown Source)
	... 35 more
Caused by: java.io.IOException: Access is denied
	at java.io.RandomAccessFile.writeBytes(Native Method)
	at java.io.RandomAccessFile.write(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.writeToLog(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.flushDirtyBuffers(Unknown Source)
	... 45 more
============= end nested exception, level (3) ===========
============= begin nested exception, level (4) ===========
ERROR XSLA8: Cannot rollback transaction 418, trying to compensate null operation with null
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.undo(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.abort(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.XactFactory.rollbackAllTransactions(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.recover(Unknown Source)
	at org.apache.derby.impl.store.raw.RawStore.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.store.access.RAMAccessManager.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.bootStore(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startProviderService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.findProviderAndStartService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startPersistentService(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.startPersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at edu.ycp.cs320.sqldemo.SQLDemo.main(SQLDemo.java:43)
Caused by: ERROR XSLA1: Log Record has been sent to the stream, but it cannot be applied to the store (Object null).  This may cause recovery problems also.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.logAndUndo(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.logAndUndo(Unknown Source)
	... 33 more
Caused by: ERROR XSLA0: Cannot flush the log file to disk H:\git\qherb-pnelson1-wwyatt-jbady\EntreLink\test.db\log\log1.dat.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseDataFileFactory.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.store.raw.data.RAFContainer.stubbify(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseContainerHandle.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerOperation.undoMe(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerUndoOperation.doMe(Unknown Source)
	... 35 more
Caused by: java.io.IOException: Access is denied
	at java.io.RandomAccessFile.writeBytes(Native Method)
	at java.io.RandomAccessFile.write(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.writeToLog(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.flushDirtyBuffers(Unknown Source)
	... 45 more
============= end nested exception, level (4) ===========
============= begin nested exception, level (5) ===========
ERROR XSLA1: Log Record has been sent to the stream, but it cannot be applied to the store (Object null).  This may cause recovery problems also.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.logAndUndo(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.logAndUndo(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.undo(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.abort(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.XactFactory.rollbackAllTransactions(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.recover(Unknown Source)
	at org.apache.derby.impl.store.raw.RawStore.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.store.access.RAMAccessManager.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.bootStore(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startProviderService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.findProviderAndStartService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startPersistentService(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.startPersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at edu.ycp.cs320.sqldemo.SQLDemo.main(SQLDemo.java:43)
Caused by: ERROR XSLA0: Cannot flush the log file to disk H:\git\qherb-pnelson1-wwyatt-jbady\EntreLink\test.db\log\log1.dat.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseDataFileFactory.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.store.raw.data.RAFContainer.stubbify(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseContainerHandle.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerOperation.undoMe(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerUndoOperation.doMe(Unknown Source)
	... 35 more
Caused by: java.io.IOException: Access is denied
	at java.io.RandomAccessFile.writeBytes(Native Method)
	at java.io.RandomAccessFile.write(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.writeToLog(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.flushDirtyBuffers(Unknown Source)
	... 45 more
============= end nested exception, level (5) ===========
============= begin nested exception, level (6) ===========
ERROR XSLA0: Cannot flush the log file to disk H:\git\qherb-pnelson1-wwyatt-jbady\EntreLink\test.db\log\log1.dat.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseDataFileFactory.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.store.raw.data.RAFContainer.stubbify(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseContainerHandle.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerOperation.undoMe(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerUndoOperation.doMe(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.logAndUndo(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.logAndUndo(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.undo(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.abort(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.XactFactory.rollbackAllTransactions(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.recover(Unknown Source)
	at org.apache.derby.impl.store.raw.RawStore.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.store.access.RAMAccessManager.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.bootStore(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startProviderService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.findProviderAndStartService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startPersistentService(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.startPersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at edu.ycp.cs320.sqldemo.SQLDemo.main(SQLDemo.java:43)
Caused by: java.io.IOException: Access is denied
	at java.io.RandomAccessFile.writeBytes(Native Method)
	at java.io.RandomAccessFile.write(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.writeToLog(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.flushDirtyBuffers(Unknown Source)
	... 45 more
============= end nested exception, level (6) ===========
============= begin nested exception, level (7) ===========
java.io.IOException: Access is denied
	at java.io.RandomAccessFile.writeBytes(Native Method)
	at java.io.RandomAccessFile.write(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.writeToLog(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogAccessFile.flushDirtyBuffers(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseDataFileFactory.flush(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.store.raw.data.RAFContainer.stubbify(Unknown Source)
	at org.apache.derby.impl.store.raw.data.RAFContainer.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.BaseContainerHandle.removeContainer(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerOperation.undoMe(Unknown Source)
	at org.apache.derby.impl.store.raw.data.ContainerUndoOperation.doMe(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.logAndUndo(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.logAndUndo(Unknown Source)
	at org.apache.derby.impl.store.raw.log.FileLogger.undo(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.Xact.abort(Unknown Source)
	at org.apache.derby.impl.store.raw.xact.XactFactory.rollbackAllTransactions(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.recover(Unknown Source)
	at org.apache.derby.impl.store.raw.RawStore.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.store.access.RAMAccessManager.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startModule(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.bootServiceModule(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.bootStore(Unknown Source)
	at org.apache.derby.impl.db.BasicDatabase.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.boot(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.bootModule(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startProviderService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.findProviderAndStartService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startPersistentService(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.startPersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at edu.ycp.cs320.sqldemo.SQLDemo.main(SQLDemo.java:43)
============= end nested exception, level (7) ===========
Cleanup action completed
